### ISSUE TITLE
Hovering @param comments duplicates the comment in the hover

### ISSUE THREAD

**Issue by Tyriar** [microsoft/TypeScript#59402]
Testing microsoft/vscode#223154

<img width="1136" alt="Screenshot 2024-07-23 at 2 07 23 PM" src="https://github.com/user-attachments/assets/0d7b65b4-ecbc-4a3c-8937-bca3421c6d00">

<img width="771" alt="Screenshot 2024-07-23 at 2 07 57 PM" src="https://github.com/user-attachments/assets/026428f6-eb04-4bd8-9199-f4b837cd73b7">

**Comment by mjbvz** [comment]
Minimal repo:

```ts
/**
 * @param x Does the thing
 */
function foo(x) {}
```

Example response after hovering over `x` in `@param x` showing the duplicated `tags` section:

```
Perf 621  [14:55:38.646] 204::quickinfo: elapsed time (in milliseconds) 5.1846
Info 622  [14:55:38.646] response:
    {"seq":0,"type":"response","command":"quickinfo","request_seq":204,"success":true,"body":{"kind":"parameter","kindModifiers":"","start":{"line":3,"offset":11},"end":{"line":3,"offset":12},"displayString":"(parameter) x: any","documentation":[{"text":"Does the thing","kind":"text"}],"tags":[{"name":"param","text":[{"text":"x","kind":"parameterName"},{"text":" ","kind":"space"},{"text":"Does the thing","kind":"text"}]}]}}
```

**Comment by a-tarasyuk** [comment]
> duplicated tags

@mjbvz It looks like there are no duplicate tags, but there is a duplicate comment from the `param` tag, which is included in both the `documentation` and the `tags` info. Should comments related to params tags be included exclusively in the tags?


/cc @sandersn

**Comment by mjbvz** [comment]
Yes, I could have been more clear: the entire `tags` section here should be omitted because we are already hovering over a specific tag. Instead we should return just tag content in the `documentation ` and leave `tags` empty
